tests/e2e/standard/contacts/create-places-contacts.wdio-spec.js

```diff
@@ -0,0 +1,44 @@
+const loginPage = require('../../../page-objects/default/login/login.wdio.page');
+const commonPage = require('../../../page-objects/default/common/common.wdio.page');
+const contactPage = require('../../../page-objects/standard/contacts/contacts.wdio.page');
+
+const PLACE_TYPES = { CLINIC: 'clinic', HEALTH_CENTER: 'health_center', DISTRICT: 'district_hospital' };
+const CONTACT_NAME_SUFFIX = ' Contact';
+
+describe('Creating contacts with standard config. ', () => {
+  const districtName = 'District test';
+  const healthCenterName = 'Health Center test';
+  const clinicName = 'Clinic test';
+
+  before(async () => {
+    await loginPage.cookieLogin();
+    await commonPage.waitForPageLoaded();
+  });
+
+  it('should create a new district hospital', async () => {
+    await commonPage.goToPeople();
+    await contactPage.addPlace(PLACE_TYPES.DISTRICT, districtName, districtName + CONTACT_NAME_SUFFIX);
+    await commonPage.waitForPageLoaded();
+    expect(await (await contactPage.contactPageDefault.contactCard()).getText()).to.equal(districtName);
+    expect(await contactPage.contactPageDefault.getPrimaryContactName()).to.equal(districtName + CONTACT_NAME_SUFFIX);
+  });
+
+  it('should create a new health center in the district hospital', async () => {
+    await commonPage.goToPeople();
+    await contactPage.contactPageDefault.selectLHSRowByText(districtName);
+    await contactPage.addPlace(PLACE_TYPES.HEALTH_CENTER, healthCenterName, healthCenterName + CONTACT_NAME_SUFFIX);
+    await commonPage.waitForPageLoaded();
+    expect(await (await contactPage.contactPageDefault.contactCard()).getText()).to.equal(healthCenterName);
```

Comment:
I think that it will be better if all the interactions with the elements happens in the `page-object/*` files. Could you please create a method in the `contactPageDefault` file that returns the text of the contactCard  and use that method in this test file?